### DISCUSSION TITLE
RabbitMQ - WebUI - Management Overview not refreshing

### DISCUSSION THREAD

**Discussion by BBQigniter** [rabbitmq/rabbitmq-server#3109] [Other] [answered]
With version 3.8.17 I noticed that there seems to be some minor issue in the webUI if in the "This time series"-dialog the "Chart range" is set to be larger or equal to "Last hour" - the stats on the overview are not being refreshed in the selected refresh-interval.
Strangely also normal refreshing via browser is not helping.

Refresh works with "Last minute" and "Last 10 minutes".

Couldn't find any info if this is now a new feature or a bug or if I missed something in changelog.

**Comment by michaelklishin** [comment, marked as answer]
Chart data is [kept for a certain period of time](https://www.rabbitmq.com/management.html#sample-retention). The interval very well may be 1 hour by default, there isn't a whole lot of context on what chart and what kind of data we're talking.

If you need longer term charts, [use Prometheus and Grafana](https://www.rabbitmq.com/prometheus.html) and not management UI.
In fact, Prometheus and Grafana is a better combination for metrics in any case.

**Reply by BBQigniter** [reply]
Woops, sorry. I must have missed that for ages :o . We are using Grafana/Prometheus too, but for a quick overview after an update or so the management overview graphs are fine too, and it was quite confusing that they were not really showing anything useful with "Last hour"+ as they did before.

Thanks for clarification!
